modules/distribution-service-aws-s3/src/main/java/org/opencastproject/distribution/aws/s3/AwsS3DistributionServiceImpl.java

```diff
@@ -108,6 +109,29 @@
   // config.properties
   public static final String OPENCAST_DOWNLOAD_URL = "org.opencastproject.download.url";
 
+  /** The load on the system introduced by creating a distribute job */
+  public static final float DEFAULT_DISTRIBUTE_JOB_LOAD = 1.0f;
```

Comment:
Should the default distribution job load actually be that high. 1 is the same as an enocde job iirc.

Reply:
The download distribution job loads do not even match up in code and configuration. I created pull request #244 to fix that.

What about 0.4 for this? Given that 1 means full usage of one CPU core, I would expect a single threaded machine to handle a bit more than two of these jobs in parallel. Of course, this is picked by me with no actual testing done. Hence, feel free to go higher or lower ;-)

Reply:
Hi @lkiesow, I did some tests yesterday and the job didn't produce as much load as I expected so I think it's ok to match to the download distribution values. I am going to set the default to 0.2 and 0.1.